src/Exceptions/HttpProxyAuthenticationRequired.php

```diff
@@ -8,4 +8,3 @@ class HttpProxyAuthenticationRequired extends HttpException {
     public const DESCRIPTION = "Is similar to 401 (Unauthorized), but indicates that the client needs to authenticate itself in order to use a proxy.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/407";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces